windows/utils/win_install_online_updates.yml

```diff
@@ -8,7 +8,6 @@
 #     Some possible categories are: Application, Connectors, Critical Updates,
 #     Definition Updates, Developer Kits, Feature Packs, Guidance, Security Updates,
 #     Service Packs, Tools, Update Rollups, Updates and Upgrades.
-#     Default value is '*', which matches all categories.
```

Comment:
Can keep the usage description for *. 
E.g. 
"Value * will match all categories."